src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdConfiguration.cs

```diff
@@ -101,6 +101,15 @@ public void Configure(string name, OpenIddictOptions options)
                 options.AccessTokenHandler = new JwtSecurityTokenHandler();
             }
 
+            if (settings.UseRollingTokens)
+            {
+                options.UseRollingTokens = true;
```

Comment:
(nit: you can save 6 lines by doing ˋoptions.UseRollingTokens = settings.UseRollingTokens;`)